### PR TITLE
feat: add footer prop to the Textarea component

### PR DESCRIPTION
**Pull request by rgah2107**
fix: #1534

## Changes proposed in this PR:
- add footer prop to the `Textarea` component

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/Textarea/index.js [hunk 1]
```diff
@@ -135,6 +137,9 @@ class Textarea extends Component {
                 <RenderIf isTrue={!!error}>
                     <StyledError id={this.getErrorMessageId()}>{error}</StyledError>
                 </RenderIf>
+                <RenderIf isTrue={!!footer}>
+                    <StyledFooter>{footer}</StyledFooter>
+                </RenderIf>
```
Comment (yvmunayev) [inline src/components/Textarea/index.js:142]: @TahimiLeonBravo the footer position is before the bottomHelpText and the error?

**Comment by rgah2107** [comment]
> 
> 
> You have to check the styles well because when you focus the component moves

File: src/components/Textarea/styled/textarea.js [hunk 2]
```diff
@@ -22,9 +22,8 @@ const StyledTextarea = attachThemeAttrs(styled.textarea)`
     :active {
         outline: 0;
         background-color: ${props => props.palette.background.main};
-        border: solid 2px ${props => props.palette.brand.main};
+        border: solid 1px ${props => props.palette.brand.main};
```
Comment (TahimiLeonBravo) [inline src/components/Textarea/styled/textarea.js:25]: we need to have the 2px border when the Textarea is active

**Comment by TahimiLeonBravo** [comment]
@rgah2107 I found some problem with border here
Screen Shot 2020-05-14 at 1 57 47 PM

I think that we should put the Textarea transparent and apply the border and background styles to the new Container

File: src/components/Textarea/styled/textarea.js (hunk 2 above)
Comment (rgah2107) [reply src/components/Textarea/styled/textarea.js:25]: ok

File: src/components/Textarea/readme.md [hunk 3]
```diff
@@ -117,3 +117,51 @@ const containerStyles = {
...
+const StyledFooter = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    font-size: 12px;
+    color: ${props => props.text.header};
+    text-align: right;
+    padding: 10px;
+    border-radius: 0 0 0.875rem 0.875rem;
+    background-color: #F6F7F9;
```
Comment (LeandroTorresSicilia) [inline src/components/Textarea/readme.md:137]: we need to use a color from theme

**Pull request merged**